tests/DoctrineDataFixtureTest/Loader/ServiceLocatorAwareLoaderTest.php

```diff
@@ -55,6 +59,9 @@ public function testLoadingFixtureWhichIsNotServiceLocatorAware()
     /**
      * Ensures that the Service Locator instance passed into the ServiceLocatorAwareLoader
      * actually makes it to the SL-aware fixtures loaded
+     *
+     * @covers DoctrineDataFixtureModule\Loader\ServiceLocatorAwareLoader::loadFromDirectory
```

Comment:
No need to be this specific: I suggest defining just class coverage at test class level, that is sufficient :-)